code/obj/ladder.dm

```diff
@@ -202,3 +202,12 @@ ADMIN_INTERACT_PROCS(/obj/ladder/embed, proc/toggle_hidden)
 
 		src.vis_contents -= proxy
 		qdel(proxy)
+
+/obj/ladder/Entered(atom/movable/AM, atom/OldLoc)
+	. = ..()
+	if(isobj(AM))
```

Comment:
why not any movable

Reply:
ex. what if i vomit up a mob i ate while climbing

Reply:
(yes, this does pose difficulties with the set loc in climb() if you change it to movables, care)

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaaaa
`var/atom/movable/proxy = new(src)`